master/custom/factories.py

```diff
@@ -125,7 +125,7 @@ def setup(self, parallel, branch, test_with_PTY=False, **kwargs):
         installed_python = "./target/bin/python%s" % branch
         self.addStep(
             Configure(
-                command=["./configure", "--prefix", "$(PWD)/target"]
+                command=["./configure", "--prefix", "${PWD}/target"]
```

Comment:
Please revert the `${PWD}` changes; `$(PWD)` is currently working as expected for us.  IIRC, I tried `${PWD}` initially and did not get the expected result.  See also my comment on closing #79.